pkgs/applications/graphics/krita/generic.nix

```diff
@@ -18,27 +19,22 @@ mkDerivation rec {
 
   src = fetchurl {
     url = "https://download.kde.org/${kde-channel}/${pname}/${version}/${pname}-${version}.tar.gz";
-    inherit sha256;
+    inherit hash;
   };
 
   patches = [
-    (fetchpatch {
-      name = "exiv2-0.28.patch";
-      url = "https://gitlab.archlinux.org/archlinux/packaging/packages/krita/-/raw/acd9a818660e86b14a66fceac295c2bab318c671/exiv2-0.28.patch";
-      hash = "sha256-iD2pyid513ThJVeotUlVDrwYANofnEiZmWINNUm/saw=";
-    })
+    ./find_openjpeg.patch
```

Comment:
Similarly to my comment re: the fftw patch - why do we need this, and can we get it upstreamed?